src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -3831,62 +3830,70 @@ namespace ConvectionCoefficients {
+        //      +---------------------+-----------+---------------------------------------------+------------------+-----------------+-------------+
+        //      | Vertical Surface    | N/A       | -0.3827 to 0.3827 (67.5 to 112.5 degrees)   | N/A              | Normal          |       3.076 |
+        //      | Horizontal Surface  | Positive  | 0.9238 to 1.0 (0 to 22.5 degrees)           | Positive         | Enhanced        |       4.043 |
+        //      | Horizontal Surface  | Positive  | -0.9238 to -1.0 (157.5 to 180 degrees)      | Negative         | Reduced         |       0.948 |
+        //      | Horizontal Surface  | Negative  | 0.9239 to 1.0 (0 to 22.5 degrees)           | Negative         | Reduced         |       0.948 |
+        //      | Horizontal Surface  | Negative  | -0.9239 to -1.0 (157.5 to 180 degrees)      | Positive         | Enhanced        |       4.040 |
+        //      | Tilted Surface      | Positive  | 0.3827 to 0.9239 (22.5 to 67.5 degrees)     | Positive         | Enhanced        |       3.870 |
+        //      | Tilted Surface      | Negative  | -0.3827 to -0.9239 (157.5 to 157.5 degrees) | Positive         | Enhanced        |       3.870 |
+        //      | Tilted Surface      | Negative  | 0.3827 to 0.9239 (22.5 to 67.5 degrees)     | Negative         | Reduced         |       2.281 |
+        //      | Tilted Surface      | Positive  | -0.3827 to -0.9239 (157.5 to 157.5 degrees) | Negative         | Reduced         |       2.281 |
+        //      +---------------------+-----------+---------------------------------------------+------------------+-----------------+-------------+
 
-        // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        Real64 DeltaTemp = Tamb - Tsurf;
 
         // Set HConvIn using the proper correlation based on DeltaTemp and Cosine of the Tilt of the Surface
-        if (std::abs(cosTilt) >= 0.9239) {   // Horizontal Surface
-            if (DeltaTemp * cosTilt < 0.0) { // Horizontal, Reduced Convection
-                return 0.948;
-            } else if (DeltaTemp * cosTilt == 0.0) { // Vertical Surface
-                return 3.076;
-            } else /*if (DeltaTemp * cosTilt > 0.0)*/ { // Horizontal, Enhanced Convection
-                return 4.040;
-            }
-        } else {                             // Tilted Surface
-            if (DeltaTemp * cosTilt < 0.0) { // Tilted, Reduced Convection
-                return 2.281;
-            } else if (DeltaTemp * cosTilt == 0.0) { // Vertical Surface
-                return 3.076;
-            } else /*if (DeltaTemp * cosTilt > 0.0)*/ { // Tilted, Enhanced Convection
-                return 3.870;
+        if (std::abs(cosTilt) < 0.3827) {  // Vertical Surface
+            return 3.076;
+        }
+        else {
+            Real64 DeltaTempCosTilt = (Tamb - Tsurf)*cosTilt;
```

Comment:
A little whitespace around that asteriscus would've been nice but not a big deal.